bip-0352/reference.py

```diff
@@ -127,6 +127,10 @@ def create_outputs(input_priv_keys: List[Tuple[ECKey, bool]], input_hash: bytes,
         negated_keys.append(k)
 
     a_sum = sum(negated_keys)
+    if not a_sum.valid:
+        # Input privkeys sum is zero -> fail
+        return []
```

Comment:
Any reason we can't reuse the private keys corresponding to the public keys in the recipient test case here?